### PR TITLE
chore(repo): increase node-max-old-space-size to avoid CI failures

### PR DESCRIPTION
**Pull request by juristr**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: scripts/e2e.ts [hunk 1]
```diff
@@ -105,17 +105,20 @@ async function runTest() {
       console.log('No tests to run');
     } else if (selectedProjects) {
       execSync(
-        `yarn nx run-many --target=e2e --projects=${selectedProjects} ${testNamePattern}`,
+        `node -max-old-space-size=4096 ./node_modules/.bin/nx run-many --target=e2e --projects=${selectedProjects} ${testNamePattern}`,
```
Comment (bekos) [inline scripts/e2e.ts:108]: I think this needs double dashes, ie `--max-old-space-size` 🤔 But I think it better to pass it as environment variable on CircleCI, as suggested [here](https://support.circleci.com/hc/en-us/articles/360009208393-How-can-I-increase-the-max-memory-for-Node-)

File: scripts/e2e.ts [hunk 2]
```diff
@@ -105,17 +105,20 @@ async function runTest() {
...
           env: { ...process.env, NX_TERMINAL_CAPTURE_STDERR: 'true' },
         }
       );
     } else {
-      execSync(`yarn nx run-many --target=e2e --all`, {
-        stdio: [0, 1, 2],
-        env: { ...process.env, NX_TERMINAL_CAPTURE_STDERR: 'true' },
-      });
+      execSync(
+        `node -max-old-space-size=4096 ./node_modules/.bin/nx run-many --target=e2e --all`,
```
Comment (jdpearce) [inline scripts/e2e.ts:116]: Aren't you missing a dash on both of these? (i.e. shouldn't it be `--max-old-space-size=4096`?)

File: scripts/e2e.ts (hunk 1 above)
Comment (juristr) [reply scripts/e2e.ts:108]: yep those should be double `--`. Probably copy & paste error from the other PR I have...but let me double check it was this that solved the issue then or the `resource_class` I had also tried before.

yeah I agree, using a ENV variable would be even better 👍 let me check

File: scripts/e2e.ts (hunk 2 above)
Comment (juristr) [reply scripts/e2e.ts:116]: yep 😓 
https://github.com/nrwl/nx/pull/5019#discussion_r593331632

**Pull request closed without merging**